node/sc/bridge_manager_test.go

```diff
@@ -131,17 +132,17 @@ func TestBridgeManager(t *testing.T) {
 		return
 	}
 
-	bridgeManager, err := NewBridgeManager(sc)
+	bm, err := NewBridgeManager(sc)
```

Comment:
@hyunsooda Please do not make unnecessary changes. You can make a separate PR for this name change.

Reply:
Reverted. Let me post another PR for reverted one later.